store/root.js

```diff
@@ -240,6 +214,28 @@ class Root extends Store {
     }
   }
 
+  @action
```

Comment:
You changed the order of the methods. It does not seem you had a reason for this, can you move this block above the `retrieveIrusStats()`?

Reply:
Actually I had a reason. changeDataProvider method calls retrieveX methods so I wanted group them together and place them directly bellow that method. :)